aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/CreateHandler.java

```diff
@@ -35,18 +36,19 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(final Amaz
                                                                           final CallbackContext callbackContext,
                                                                           final ProxyClient<RdsClient> proxyClient,
                                                                           final Logger logger) {
-        final Tagging.TagSet systemTags = Tagging.TagSet.builder()
-                .systemTags(Tagging.translateTagsToSdk(request.getSystemTags()))
-                .build();
 
         final Tagging.TagSet extraTags = Tagging.TagSet.builder()
                 .stackTags(Tagging.translateTagsToSdk(request.getDesiredResourceTags()))
                 .resourceTags(Translator.translateTagsToSdk(request.getDesiredResourceState().getTags()))
                 .build();
 
+        final Tagging.TagSet allTags = extraTags.toBuilder()
+                .systemTags(Tagging.translateTagsToSdk(request.getSystemTags()))
+                .build();
+
```

Comment:
nit: I would suggest flipping it the other way around: all tags is the primary declaration and the extra tags as a subset.

Reply:
Done!